e3sm_diags/driver/utils/dataset_xr.py

```diff
@@ -163,7 +163,9 @@ def __init__(
 
     @property
     def is_time_series(self):
-        if self.parameter.ref_timeseries_input or self.parameter.test_timeseries_input:
+        if (self.data_type == "ref" and self.parameter.ref_timeseries_input) or (
+            self.data_type == "test" and self.parameter.test_timeseries_input
+        ):
             return True
         else:
             return False
```

Comment:
A more explicit version of the `main` code below:
https://github.com/E3SM-Project/e3sm_diags/blob/4cda0a807e496a97392c14d92429bd614c536e4d/e3sm_diags/driver/utils/dataset.py#L188-L195